src/icalendar/parser_tools.py

```diff
@@ -21,7 +22,7 @@ def from_unicode(value: Any, encoding='utf-8') -> bytes:
     return value
 
 
-def to_unicode(value, encoding='utf-8'):
+def to_unicode(value:ICAL_TYPE, encoding='utf-8') -> str:
```

Comment:
I would expect a linter to catch space inconsistencies.

```suggestion
def to_unicode(value: ICAL_TYPE, encoding='utf-8') -> str:
```